libs/payments/ui/src/lib/client/components/PurchaseDetails/index.tsx

```diff
@@ -0,0 +1,92 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use client';
+
+import { Localized } from '@fluent/react';
+import Image from 'next/image';
+import { useState } from 'react';
+import chevron from './images/chevron.svg';
+
+type PurchaseDetailsProps = {
+  children: React.ReactNode;
+  priceInterval: React.ReactNode;
+  purchaseDetails: {
+    details: string[];
+    subtitle: string | null;
+    productName: string;
+    webIcon: string;
+  };
+};
+
+export function PurchaseDetails(props: PurchaseDetailsProps) {
+  const { children, priceInterval, purchaseDetails } = props;
+  const { subtitle, productName, webIcon } = purchaseDetails;
+  const [detailsHidden, setDetailsState] = useState(false);
```

Comment:
Could this be flipped to `true`? On mobile view the purchase details should start hidden.

Reply:
Aah, I did not realize that. I'll make this update, thanks!